lib/presentation/resources/assets_manager.dart

```diff
@@ -1,4 +1,15 @@
 const String IMAGES_PATH = "assets/images";
 const String ICONS_PATH = "assets/icons";
 
-class AssetsManager {}
+class AssetsManager {
+  static CustomIcons icons = CustomIcons();
```

Comment:
@dilumdesilva why do we need static references of classes inside. We can directly have static constants inside AssetsManager.

Reply:
We actually don't need asset manager we need something like Flutter SDK implementation. Colors.red, Icons.star

Ours were like AssetManger.delete doesn't make sense that's why it was re-organised AssetManager.icons.delete